lib/MusicBrainz/Server.pm

```diff
@@ -471,6 +471,10 @@ sub TO_JSON {
         packagings
         place_types
         primary_types
+        release_format_stats
+        release_status_stats
+        release_type_stats
```

Comment:
It seems unneeded to pass it through the stash as it is passed as prop `releaseTypeStats` already.

The same holds for most (all?) of stash additions in this pull request.